modules/field_event_dispatcher/field_event_dispatcher.services.yml

```diff
@@ -3,3 +3,8 @@ services:
     class: Drupal\field_event_dispatcher\EventSubscriber\Form\FormEntityViewDisplayEditAlterSubscriber
     tags:
       - { name: 'event_subscriber' }
+field_event_dispatcher.form_entity_form_display_edit_alter_event_subscriber:
```

Comment:
Please add a new line between service defs.
Add missing 2 spaces before this.
Update the naming with the above naming as an example.